#EXERCISE 1
Python 3.6.3 (v3.6.3:2c5fed8, Oct  3 2017, 18:11:49) [MSC v.1900 64 bit (AMD64)]
 on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> n = 42
>>> 42 = n
  File "<stdin>", line 1
SyntaxError: can't assign to literal
>>> x = y = 1
>>> x = 2;
>>> xy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'xy' is not defined

#Exercise 2.1 Volume of a Sphere
>>> import math
>>> (4/3) * math.pi * 5^3
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for ^: 'float' and 'int'
>>> (4/3) * math.pi * math.pow(5, 3)
523.5987755982989

#Exercise 2.2 Cost of copies
>>> (24.95 * .6) * 60 + (.75 * 59) + 3
945.4499999999999

#Exercise 2.3 Arrive home after run
# 2 miles at easy pace, plus 3 at tempo
>>> (8 + 15/60) * 2 + (7 + 12/60) * 3
38.1
>>> (8 + 15/60) * 2 + (7 + 12/60) * 3
38.1 #Total Run Time
>>>
#Leave at 6:52am and arrive 38:06 later at 7:30 pm and 6 seconds

>>> from datetime import datetime
>>> dt = datetime.datetime(2017, 10,24, 6, 52)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'datetime.datetime' has no attribute 'datetime'
>>> dt = datetime(2017, 10,24, 6, 52)
>>> delta = timedelta(mintues=38)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'timedelta' is not defined
>>> from datetime import timedelta
>>> delta = timedelta(mintues=38)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'mintues' is an invalid keyword argument for this function
>>> delta = timedelta(minutes=38)
>>> newtime = dt + delta
>>> print(newtime)
2017-10-24 07:30:00
>>>
KeyboardInterrupt
>>> import datetime
>>> tm = datetime.time(6, 52)
>>> tm + delta
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'datetime.time' and 'datetime.timedelta'
>>> newtime = tm + delta
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'datetime.time' and 'datetime.timedelta'
>>> oldtime = time(6, 52)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'time' is not defined
>>> from datetime import time
>>> oldtime = time(6,52)
>>>

#from x import y - don't need to do x.y, just y()
# import x - now I need to qualify y with x like x.y()
